.github/workflows/schemaspy.yml

```diff
@@ -34,14 +34,18 @@ jobs:
           npm install
           npm run alcs:migration:run
         working-directory: services
+      - name: Import OATS
```

Comment:
Looks like this will run every time we generate schemaspy, is that necessary?

Reply:
removed!